generate_compose/odcs_requester.py

```diff
@@ -16,9 +19,21 @@ class ODCSRequestReference(ComposeReference):
 @dataclass(frozen=True)
 class ODCSRequester(ComposeRequester):
     """
-    Request a new ODCS compose based on compose configurations and return a reference
+    Request a new ODCS composes based on compose configurations and return a reference
     to the remote compose location.
     """
 
-    def __call__(self, configs: ComposeConfigurations) -> ODCSRequestReference:
-        raise NotImplementedError()
+    odcs: ODCS = ODCS("https://odcs.engineering.redhat.com/")
+
+    def __call__(self, compose_configs: ComposeConfigurations) -> ODCSRequestReferences:
+        assert isinstance(compose_configs, ODCSComposesConfigs)
+        composes = [
+            self.odcs.request_compose(config.spec, **config.additional_args)
```

Comment:
we would need a followup pr to add some calls to a logger to show what's is going on